v2_jobs_test.go

```diff
@@ -36,6 +42,11 @@ func TestJobs_List(t *testing.T) {
 	})
 
 	assert.NilError(t, err)
+	ti, _ := time.Parse("2006-01-02T15:04:05", "2024-01-18T01:27:49")
+
+	var tdt TaskDateTime
+	tdt.Time = ti
+
```

Comment:
In this test if I understand correctly `JobsListRequest` should have `Detailed` set to `true`, right?

Ideally we would have two subtests, with Detailed set to true and false, confirming that the fields are set to their zero values.

Reply:
This is a good catch I can add this. That said I would like to add that in a separate PR building on this one.